### PR TITLE
W-14990410-securesession-linksterminalserversoftware-se

### PR DESCRIPTION
**Pull request by sebermuley**
Added information about Secure Sessions:
- examples for terminal server software
- how to compute the needed RAM

### CODE REVIEW

**Review by oveltan — DISMISSED** [review]
Great work!

**Review by oveltan — DISMISSED** [review]
Great!

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 1]
```diff
@@ -1,7 +1,18 @@
 == Hardware Requirements
 
 * For optimal performance, use a dedicated computer for running the RPA Bot.
-
+* The following table shows the minimum and the recommended hardware requirements for running one process. To run more than one process in parallel, when using Secure Sessions, you need to add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
```
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: Should this bullet be written more as a requirement with the details below?

* Use a computer with the required amount of RAM.
   The amount of RAM required depends on the number of processes you plan to run in parallel.
   
  Follow this with the table intro and the table. Then include the example with the formula.

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
++
+For example, let's say you want to run two different processes in parallel. The first process requires 1 GB RAM and should run in ten times in parallel. The second process requires 3 GB RAM and should run in five times in parallel. Then you need a computer with a minimum RAM of 63 GB:
```
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: Can you streamline this a little? Maybe something like:

For example, to run two processes in parallel:

* Process 1 requires one GB of RAM and runs 10 times in parallel.
* Process 2 requires three GB of RAM and runs 5 times in parallel.

In this case, use the following formula to calculate that you need a computer with a minimum of 63 GB of RAM:

Comment (hannanelson) [reply rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: The [style for numbers](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-numbers) is to spell out 1-9 and use digits for 10+

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 3]
```diff
@@ -50,12 +61,18 @@ To use Secure Sessions you must ensure that all prerequisites are met and all re
 ** Use dedicated accounts and a user-naming scheme to differentiate between local users and domain users and between human users and bot users.
 ** User accounts used by the bot to run processes must belong to the `Remote Desktop Users` group and must have permission to establish a local RDP session via the local loopback address 127.0.0.2.
 * To enable a bot to run more than one (Windows 10, Windows 11) or two (Windows Server) Secure Sessions simultaneously, ensure that a terminal server software with enough licensed users is installed on the computer where the bot is installed. The computer must have enough resources to execute several processes simultaneously: 2 GB or more RAM per session.
++
+
+Among others, you can use one of the following terminal server software:
```
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: Is there a way to say that these have been tested and confirmed to work?

Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]:
```suggestion
Although other terminal servers might work, RPA has been validated with the following terminal servers:
```

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc (hunk 1 above)
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: Move the intro to the table to immediately before the table.

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 4]
```diff
@@ -50,12 +72,18 @@ To use Secure Sessions you must ensure that all prerequisites are met and all re
 ** Use dedicated accounts and a user-naming scheme to differentiate between local users and domain users and between human users and bot users.
 ** User accounts used by the bot to run processes must belong to the `Remote Desktop Users` group and must have permission to establish a local RDP session via the local loopback address 127.0.0.2.
 * To enable a bot to run more than one (Windows 10, Windows 11) or two (Windows Server) Secure Sessions simultaneously, ensure that a terminal server software with enough licensed users is installed on the computer where the bot is installed. The computer must have enough resources to execute several processes simultaneously: 2 GB or more RAM per session.
++
+
+Although other terminal server software might work, RPA has been validated with the following terminal server software:
```
Comment (Crispy-Salesforce) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]:
```suggestion
Although other terminal server software can work, RPA has been validated with the following terminal server software:
```

**Review by Crispy-Salesforce — DISMISSED** [review]

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 5]
```diff
@@ -10,6 +12,26 @@
 | 30 GB of disk space | 30 GB (SSD) of disk space
 |===
 
+* To run more than one process in parallel, when using Secure Sessions, you need to add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
++
+For example, to run two different processes in parallel:
++
+Process 1 requires one GB of RAM and runs 10 times in parallel.
```
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: Suggest making the Process sentences subbullets (if you leave the bullet on line 15).
```
+
--
** Process 1 ...
** Process 2 ...
--
```

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 6]
```diff
@@ -10,6 +12,26 @@
...
++
+Process 2 requires three GB of RAM and runs 5 times in parallel.
```
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]:
```suggestion
Process 2 requires three GB of RAM and runs five times in parallel.
```

Comment (hannanelson) [reply rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: I gave you bad advice by calling these "Process 1". Maybe Process A and Process B?

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 7]
```diff
@@ -1,7 +1,9 @@
 == Hardware Requirements
 
 * For optimal performance, use a dedicated computer for running the RPA Bot.
-
+* Use a computer with the required amount of RAM. The amount of RAM required depends on the number of processes to run in parallel.
+* The following table shows the minimum and the recommended hardware requirements for running one process:
```
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc:5]: I think this should be a paragraph under the bullet.

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 8]
```diff
@@ -10,6 +12,26 @@
 | 30 GB of disk space | 30 GB (SSD) of disk space
 |===
 
+* To run more than one process in parallel, when using Secure Sessions, you need to add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
```
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: I think this should be a paragraph, not a bullet.

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc (hunk 5 above)
Comment (sebermuley) [reply rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: Thanks for this tip with the -- I tried in vain to format it this way :)

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 9]
```diff
@@ -10,6 +13,24 @@
 | 30 GB of disk space | 30 GB (SSD) of disk space
 |===
 
+To run more than one process in parallel, when using Secure Sessions, you need to add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
```
Comment (Crispy-Salesforce) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]:
```suggestion
To run more than one process in parallel, when using Secure Sessions, you must add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
```

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by Crispy-Salesforce — DISMISSED** [review]

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 10]
```diff
@@ -10,6 +13,24 @@
 | 30 GB of disk space | 30 GB (SSD) of disk space
 |===
 
+To run more than one process in parallel, when using Secure Sessions, you must add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
+
+For example, to run two different processes in parallel:
+
+* Process A requires one GB of RAM and runs 10 times in parallel.
+* Process 2 requires three GB of RAM and runs five times in parallel.
```
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc:21]:
```suggestion
* Process B requires three GB of RAM and runs five times in parallel.
```

Comment (hannanelson) [reply rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc:21]: But now I see the problem with naming them alphabetically, it doesn't work with the P_n_ in the formula 😭 so maybe go back to Process 1 and Process 2?

File: rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc [hunk 11]
```diff
@@ -1,6 +1,9 @@
 == Hardware Requirements
 
 * For optimal performance, use a dedicated computer for running the RPA Bot.
+* Use a computer with the required amount of RAM. The amount of RAM required depends on the number of processes to run in parallel.
+
```
Comment (hannanelson) [inline rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc]: If you want to make it a hanging paragraph, add `--` after the `+` on line 5 and a closing `--` after the `++++` on line 32.

**Review by hannanelson — APPROVED** [review]

**Pull request merged**